pyproject.toml

```diff
@@ -32,6 +32,14 @@ boto3 = "*"
 minio = "^7.2.7"
 gevent = "^24.2.1"
 pyarrow = "16.1.0" # transitive dependency, pinned here to avoid conflicts
+openai = "*"
+python-docx = "*"
+PyPDF2 = "*"
+beautifulsoup4 = "*"
+pylatexenc = "*"
+python-pptx = "*"
+xlrd = "*"
+opencv-python = "*"
```

Comment:
Can you remove these dependencies, now you have moved them to the test section?

Reply:
removed.